### PR TITLE
Added XmlExtensions Unit Tests.

### PR DESCRIPTION
**Pull request by michelvosje**
Implemented the XmlExtensionsUnitTests class for ticket #14.

### CODE REVIEW

File: Source/MoreDotNet.Test/Extensions/Common/XmlExtensions/XmlExtensionsTests.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+            Assert.Throws<ArgumentNullException>(() => testObject.XmlSerialize());
+        }
+
+        [Fact]
+        public void XmlSerialize_GivenNonNullArgument_ShouldReturnSerialized()
+        {
+            object testObject = new object();
+
+            Assert.NotNull(testObject.XmlSerialize());
```
Comment (Teodor92) [inline Source/MoreDotNet.Test/Extensions/Common/XmlExtensions/XmlExtensionsTests.cs]: This just checks if the returned string is not null, but is it truly correctly serialized? An additional check will make the test better.

File: Source/MoreDotNet.Test/Extensions/Common/XmlExtensions/XmlExtensionsTests.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            Assert.Throws<ArgumentNullException>(() => testSerializedObject.XmlDeserialize<object>());
+        }
+
+        [Fact]
+        public void XmlDeserialize_GivenNonNullArgument_ShouldReturnDeserialized()
+        {
+            object testObject = new object();
+            string testSerializedObject = testObject.XmlSerialize();
+
+            Assert.NotNull(testSerializedObject.XmlDeserialize<object>());
```
Comment (Teodor92) [inline Source/MoreDotNet.Test/Extensions/Common/XmlExtensions/XmlExtensionsTests.cs]: Same as above - it necessary to check if the deserialization is correct.

File: Source/MoreDotNet.Test/MoreDotNet.Tests.csproj [hunk 3]
```diff
@@ -40,9 +40,13 @@
     <CodeAnalysisRuleSet>MoreDotNet.Tests.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <Private>False</Private>
+    </Reference>
```
Comment (Teodor92) [inline Source/MoreDotNet.Test/MoreDotNet.Tests.csproj]: Please remove the reference to Microsoft.VisualStudio.QualityTools.UnitTestFramework.

**Review by Teodor92 — CHANGES_REQUESTED** [review]
Have in mind that there are a few test cases missing. For example:
- What happens if you try to deserialize to an incorrect type?

Anyway if you fix the few remarks I made in the comments, I will merge the PR :)

**Comment by michelvosje** [comment]
Yes you are right. Got a little bit short on time. Will fix it tomorrow.

**Comment by michelvosje** [comment]
I fixed the issues. What now?

**Review by Teodor92 — APPROVED** [review]

**Comment by Teodor92** [comment]
Merged!

**Comment by michelvosje** [comment]
Doing even small tickets like these give me more experience with git, Github and how to contribute to open projects.
Ty.

**Pull request merged**